src/components/Product/ListProduct.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import { FirestoreCollection } from 'react-firestore';
+import { differenceInDays } from 'date-fns';
+
 const ListProduct = () => {
+  const lastDate = new Date();
+  const currentDateSeconds = lastDate.getTime() / 1000;
```

Comment:
👍 Nice name for this variable. Good work,

Reply:
thanks